apiVersion: openapi/v1alpha1
kind: Radar
metadata:
    name: "22047995"
    labels:
        datastore_id: "4833948580446208"
data:
    classification: Serious Bug
    created: "2015-07-29T09:11:29.91031Z"
    description: |-
        Summary:

        I’m running the tests of my project, code can be found here: https://github.com/contentful/contentful.objc

        The Core Data part of the test suite has been working fine for iOS 6, 7 and 8, but when running the tests using Xcode 7 / iOS 9 in the iOS simulator, I’m receiving an “SQLite error code:522, 'not an error’”. You can find the full console output attached to this bug report.

        More specifically, the failing test is “-[CoreDataBasicTests testContinueSyncWithSameManager]” — it fails every time and the error is quite non-descriptive.

        Steps to Reproduce:

        1. Run the test suite of https://github.com/contentful/contentful.objc

        Expected Results:

        The test suite passes, as it does on iOS 8 using Xcode 6.4

        Actual Results:

        The test “-[CoreDataBasicTests testContinueSyncWithSameManager]” fails because of “SQLite error code:522, 'not an error’”.

        Regression:

        This has been working fine for multiple iOS releases, as described above.

        Notes:

        I’m open to the possibility that this is my fault, but the error message needs to be a lot more descriptive.
    email: neocool2@googlemail.com
    modified: "2015-07-29T09:11:29.91053Z"
    number: "22047995"
    number_intvalue: 22047995
    originated: 29-Jul-2015 11:11 AM
    parent_number: '&{NULL_VALUE}'
    product: iOS SDK
    product_version: ""
    reproducible: Always
    resolved: ""
    status: Open
    title: 'Core Data: executeFetchRequest:error: leads to SQLite error code:522, ''not an error'''
